### DISCUSSION TITLE
Is there a plugin like plugin-styled-components for pandacss?

### DISCUSSION THREAD

**Discussion by omerman** [chakra-ui/panda#1381] [Q&A] [answered]
Currently when using the jsx styled() factory to create components,
they don't have displayName prop in dev mode.

I was wondering if there is a plugin that could help with that so we could have better Debugging experience.

Thank you 🙏

**Comment by astahmer** [comment, marked as answer]
you should have a displayName most of the time already but this PR fixes some cases
https://github.com/chakra-ui/panda/pull/1386

**Reply by omerman** [reply]
Oh that's great ill be sure to check and report back
